legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/auth/GitLabAuthorizer.java

```diff
@@ -15,9 +15,9 @@
 package org.finos.legend.sdlc.server.gitlab.auth;
 
 import org.finos.legend.sdlc.server.auth.Session;
-import org.finos.legend.sdlc.server.gitlab.mode.GitLabModeInfo;
+import org.finos.legend.sdlc.server.gitlab.GitLabAppInfo;
 
 public interface GitLabAuthorizer
 {
-    GitLabToken authorize(Session session, GitLabModeInfo modeInfo);
+    GitLabToken authorize(Session session, GitLabAppInfo appInfo);
```

Comment:
I don't really want to keep `GitLabModeInfo` around just because of this. Plus I think `appInfo` makes sense and unless we do something like the following:

```java

default GitLabToken authorize(Session session, GitLabAppInfo appInfo)
{
    return this.authorize(Session session, <create a mock modeInfo object here>);
}
```

Which is a bit less than ideal ...